src/controllers/bookingForm.js

```diff
@@ -44,5 +43,5 @@ exports.post = (async (req, res) => {
       html: html_email,
     });
   }
-  await main().catch((e) => console.log(1111111, e));
+  await main().catch(res.status(200).send({ msg: 'done' }));
```

Comment:
I think this is a typo? shouldn't it trigger the 
![image](https://user-images.githubusercontent.com/39469813/66996850-549a4580-f0c9-11e9-9675-eb467c6d0ddb.png) in the domBoking.js